README.md

```diff
@@ -1,8 +1,11 @@
+
 # Moltin Swift Example App
 A simple example app that demonstrates how to use the Moltin iOS eCommerce SDK in Swift.
 
 *Prefer Objective-C? [Check out our Objective-C example app instead](https://github.com/moltin/ios-objc-example)*
 
+**Please note that this example is applicable to the deprecated V1 moltin API only. For the most up to date example, go to https://github.com/moltin/ios-sdk/tree/v2/Example**
```

Comment:
Move right to the top of the readme

Reply:
Done 0b922f15aa407e73296df574145f6907a8ebc20e